packages/obsolete-datx-jsonapi-angular/package.json

```diff
@@ -23,22 +23,23 @@
     "angular"
   ],
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^17.1.0",
-    "@rollup/plugin-node-resolve": "^11.1.1",
-    "@types/jest": "^26.0.20",
+    "@rollup/plugin-commonjs": "^21.0.0",
+    "@rollup/plugin-node-resolve": "^13.0.0",
+    "@rollup/plugin-typescript": "^8.2.1",
+    "@types/jest": "^27.0.0",
     "@types/lodash": "^4.14.168",
-    "@types/node": "^14.14.27",
+    "@types/node": "^16.0.0",
     "@types/uuid": "^8.3.0",
     "isomorphic-fetch": "^3.0.0",
-    "jest": "^26.6.3",
+    "jest": "^27.0.5",
     "lodash": "^4.17.20",
     "rollup": "^2.38.5",
     "rollup-plugin-exclude-dependencies-from-bundle": "^1.1.17",
     "rollup-plugin-terser": "^7.0.2",
-    "rollup-plugin-typescript2": "^0.29.0",
-    "rxjs": "^6.6.3",
-    "ts-jest": "^26.5.1",
-    "typescript": "^4.1.5",
+    "rxjs": "^7.0.0",
```

Comment:
TODO: check the whole project how we define peer and dev deps for RxJS. Angular >=12.2 works with RxJS 7 but older versions might not